code/drasil-lang/lib/Language/Drasil/Expr/Class.hs

```diff
@@ -114,7 +116,8 @@ class ExprC r where
   ($=>), ($<=>) :: r -> r -> r
   
   ($&&), ($||) :: r -> r -> r
-  
+
+  in' :: r -> r -> r
```

Comment:
This also needs a comment. I think I know what it means (it is the set-theoretic membership relation), and I think I understand why it has a funny name (else it's a conflict with Haskell), but still, this should all be documented.